sorc/ncep_post.fd/WRFPOST.F

```diff
@@ -423,11 +424,12 @@ PROGRAM WRFPOST
           Status=nf90_get_att(ncid2d,nf90_global,'landsfcmdl', iSF_SURFACE_PHYSICS)
           if(Status/=0)then
             print*,'landsfcmdl not found; assigning to 2'
-            iSF_SURFACE_PHYSICS=2 !set LSM physics to 2 for NOAH
-          endif
-          if(iSF_SURFACE_PHYSICS<2)then
-            iSF_SURFACE_PHYSICS=2 !set LSM physics to 2 for NOAH
+            iSF_SURFACE_PHYSICS=2 !set LSM physics to 2 for NOAH MP
+            ! LSM physics 1 for NOAH
           endif
+!          if(iSF_SURFACE_PHYSICS<2)then
+!            iSF_SURFACE_PHYSICS=2 !set LSM physics to 2 for NOAH
```

Comment:
I think this block supports GFS, FV3R, and RAP/HRRR, right?  I believe RAP/HRRR still uses RUC.

Reply:
@WenMeng-NOAA thank you for explanation.  I will approve next